add-on/chrome/bootstrap.js

```diff
@@ -926,6 +1026,7 @@ var WindowListener = {
     XPCOMUtils.defineLazyModuleGetter(LoopUI, "PlacesUtils", "resource://gre/modules/PlacesUtils.jsm");
 
     LoopUI.init();
+    window.LoopThrottler = Throttler;
     window.LoopUI = LoopUI;
```

Comment:
If LoopThrottler is only exported for testing, please be explicit about that here.